synapse/lib/cell.py

```diff
@@ -2595,6 +2595,9 @@ def getUserName(self, iden, defv='<unknown>'):
             return defv
         return user.name
 
+    async def hasHttpSess(self, iden):
+        return bool(await self.getHttpSessDict(iden))
```

Comment:
A sess dict that is empty would return False here. Is that possible/intended?

Reply:
It shouldn't be possible unless someone has popped the `created` value out of the session. I've added a method on the guidstor to do an existence check and am using that.